README.md

```diff
@@ -30,7 +30,7 @@ import convertHtmlToReact from '@hedgedoc/html-to-react';
 class HtmlComponent extends React.Component {
   render() {
     const html = '<div>Example HTML string</div>';
-    return <div>{ ReactHtmlParser(html) }</div>;
+    return <div>{ convertHtmlToReact (html, true) }</div>;
```

Comment:
- The commit message isn't correct. You don't use the package name here.
- no space after the name please.
- why the `true`? The options are wrapped in an object.